### PR TITLE
v1/accountpublic: Add account public API

### PR DESCRIPTION
**Pull request by nkristianto**
Add account public api definition.

### CODE REVIEW

File: v1/accountpublic/accountpublic.gunk [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+type GetPublicAccountByIDResponse struct {
+	ID              string          `pb:"1" json:"id"`
+	BankID          string          `pb:"2" json:"bank_id"`
+	Label           string          `pb:"3" json:"label"`
+	Number          string          `pb:"4" json:"number"`
+	Owners          Owner           `pb:"5" json:"owners"`
+	Type            string          `pb:"6" json:"type"`
+	Balance         types.Amount    `pb:"7" json:"balance"`
+	AccountRoutings []AccountRating `pb:"8" json:"account_routings"`
```
Comment (jwangsadinata) [inline v1/accountpublic/accountpublic.gunk]: should this be Routings or Ratings?

Comment (jwangsadinata) [reply v1/accountpublic/accountpublic.gunk]: I think let's change it if necessary.

**Review by jwangsadinata — APPROVED** [review]
One minor comment, but other than that LGTM.

File: v1/accountpublic/accountpublic.gunk (hunk 1 above)
Comment (nkristianto) [reply v1/accountpublic/accountpublic.gunk]: ups, sorry my bad

**Comment by nkristianto** [comment]
@jwangsadinata updated, PTAL

**Review by jwangsadinata — APPROVED** [review]

**Pull request merged**